interface-definitions/interfaces_wireless.xml.in

```diff
@@ -428,6 +416,133 @@
                   </leafNode>
                 </children>
               </node>
+              <leafNode name="require-vht">
+                <properties>
+                  <help>Require stations to support VHT PHY</help>
+                  <valueless/>
+                </properties>
+              </leafNode>
+              <node name="he">
+                <properties>
+                  <help>HE (High Efficiency) settings</help>
```

Comment:
High Efficiency (HE) settings

Reply:
I write it that way to be in-line with [line 30](https://github.com/vyos/vyos-1x/blob/4ff64d4a73e84cacad1db3029854d846f94b1b6e/interface-definitions/interfaces_wireless.xml.in#L30) and [line 193](https://github.com/vyos/vyos-1x/blob/4ff64d4a73e84cacad1db3029854d846f94b1b6e/interface-definitions/interfaces_wireless.xml.in#L193) which were already in there for a long time.

Reply:
Why not rewrite it to the correct form?
You can check other commands and examples
There are examples:
```
# set protocols 
Possible completions:
 > babel                Babel Routing Protocol
 > bfd                  Bidirectional Forwarding Detection (BFD)
 > bgp                  Border Gateway Protocol (BGP)
 > failover             Failover Routing
 > igmp-proxy           Internet Group Management Protocol (IGMP) proxy parameters
 > isis                 Intermediate System to Intermediate System (IS-IS)
 > mpls                 Multiprotocol Label Switching (MPLS)
 > nhrp                 Next Hop Resolution Protocol (NHRP) parameters
 > ospf                 Open Shortest Path First (OSPF)
 > ospfv3               Open Shortest Path First (OSPF) for IPv6
 > pim                  Protocol Independent Multicast (PIM) and IGMP
 > pim6                 Protocol Independent Multicast for IPv6 (PIMv6) and MLD
 > rip                  Routing Information Protocol (RIP) parameters
 > ripng                Routing Information Protocol (RIPng) parameters
 > rpki                 Resource Public Key Infrastructure (RPKI)
```
another:
```
# set 
Possible completions:
...
 > nat                  Network Address Translation (NAT) parameters
 > nat64                Network Address Translation (NAT64) parameters
 > nat66                Network Prefix Translation (NAT66/NPTv6) parameters
 > pki                  Public key infrastructure (PKI)
...
 > qos                  Quality of Service (QoS)
...
 > vpn                  Virtual Private Network (VPN)
```